wetmap/src/components/newModals/diveSiteImage/.stories.tsx

```diff
@@ -0,0 +1,92 @@
+// More on how to set up stories at: https://storybook.js.org/docs/writing-stories#default-export
+import React from 'react';
+import type { Meta, StoryObj } from '@storybook/react';
+
+import DiveSiteImageView from './view';
+
+const pic = {
+  id:           1,
+  label:        'Test',
+  UserName:     'Tester',
+  photoFile:    'https://pub-c089cae46f7047e498ea7f80125058d5.r2.dev/1731590460950.jpg',
+  commentcount: 0,
+};
+
+const meta: Meta<typeof DiveSiteImageView> = {
+  title:      'Components/Modals/DiveSiteImage',
+  component:  DiveSiteImageView,
+  tags:       ['autodocs'],
+  args:       { pic, countOfLikes: 0, picLiked: false },
+  decorators: [
+    Story => (
+      <div style={{ width: '500px' }}>
+        <Story />
+      </div>
+    ),
+  ],
+};
+
+export default meta;
+type Story = StoryObj<typeof DiveSiteImageView>;
+
+
+export const Image: Story = {
+};
+
```

Comment:
I've added few stories to your component. I think it should be helpful.
Please try to run `npm run storybook` to check how it looks in different cases. It would be nice if you add other stories with other combination of properties, different edge cases, etc.